contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -99,8 +91,11 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
     }
 
     // Add recipient to the allow list
-    s_allowedSenders.add(recipient);
-    emit AddedAccess(recipient);
+    bool success = s_allowedSenders.add(recipient);
```

Comment:
Rename "success" to "newAllowedSenderAdded" (or something descriptive like that). Then the code is self-explanatory and the extra comment below is not needed any more.